### ISSUE TITLE
Update for cuBLASMp 0.3.0

### ISSUE THREAD

**Issue by adibbley** [conda-forge/cublasmp-feedstock#3]
This issue tracks incoming PRs updating to 0.3.0

**Comment by mrogowski** [comment]
@adibbley this release adds the dependency on NVSHMEM. I think we will need to wait until they are on conda?

**Comment by adibbley** [comment]
Yeah no problem waiting, these are just created as part of cublasmp's release.

I do see the expected error in the log of https://github.com/conda-forge/libcublasmp-feedstock/pull/4
`ERROR (libcublasmp,lib/libcublasmp.so.0.3.0): $RPATH/libnvshmem_host.so.3 not found in packages, sysroot(s) nor the missing_dso_whitelist.`

**Comment by jakirkham** [comment]
Added an issue to track adding `libnvshmem` as a Conda package: https://github.com/conda-forge/staged-recipes/issues/28111

**Comment by sisodia1701** [comment]
Kirril to follow-up if `libnvshem `is a hard dependency here or not.

**Comment by mrogowski** [comment]
NVSHMEM is a hard dependency since cuBLASMp 0.3.0 release.

**Comment by jakirkham** [comment]
cc @kvoronin

**Comment by kvoronin** [comment]
To have it written here: yes, nvshmem is a hard dependency.
